From b630f4671a05d200a692d2e6635124230bda2a95 Mon Sep 17 00:00:00 2001
From: Christian Hohnstaedt <Christian.Hohnstaedt@wago.com>
Date: Tue, 19 May 2020 07:11:00 +0200
Subject: [PATCH] Fixup include file after ptn5150 driver backport

commit 176aa3601213 ("extcon: Split out extcon header
	file for consumer and provider device")'

did split the include files after v4.14-rc1-1-g176aa3601213

Signed-off-by: Christian Hohnstaedt <Christian.Hohnstaedt@wago.com>
---
 drivers/extcon/extcon-ptn5150.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/extcon/extcon-ptn5150.c b/drivers/extcon/extcon-ptn5150.c
index d1c997599390..1e1be240e551 100644
--- a/drivers/extcon/extcon-ptn5150.c
+++ b/drivers/extcon/extcon-ptn5150.c
@@ -13,7 +13,7 @@
 #include <linux/module.h>
 #include <linux/regmap.h>
 #include <linux/slab.h>
-#include <linux/extcon-provider.h>
+#include <linux/extcon.h>
 #include <linux/gpio/consumer.h>
 
 /* PTN5150 registers */
-- 
2.17.1

